src/app/[locale]/(default)/news/[article]/page.tsx

```diff
@@ -85,7 +85,7 @@ export default function ArticlePage({
         </div>
         <Badge variant='secondary'>{`${article.views} ${t('views')}`}</Badge>
       </section>
```

Comment:
should have mt-6

Reply:
Does not have any effect as space-y-4 is set on parent.